eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -1,20 +1,31 @@
 {
+  "displayNames": {
+    "--disablecov": ""
+  },
   "matrix": {
     "Agent": {
-      "ubuntu-18.04": { "OSVmImage": "MMSUbuntu18.04", "Pool": "azsdk-pool-mms-ubuntu-1804-general" },
-      "windows-2019": { "OSVmImage": "MMS2019", "Pool": "azsdk-pool-mms-win-2019-general" },
-      "macOS-10.15": { "OSVmImage": "macOS-10.15", "Pool": "Azure Pipelines" }
+      "ubuntu-18.04": { "OSVmImage": "MMSUbuntu18.04", "Pool": "azsdk-pool-mms-ubuntu-1804-general" }
     },
-    "PythonVersion": [ "3.5", "3.7", "2.7", "pypy3" ],
-    "CoverageArg": [ "--disablecov" ]
+    "PythonVersion": [ "3.7", "3.8", "3.9", "pypy3" ],
+    "CoverageArg": "--disablecov"
   },
   "include": [
     {
       "Agent": {
-          "ubuntu-18.04": { "OSVmImage": "MMSUbuntu18.04", "Pool": "azsdk-pool-mms-ubuntu-1804-general" }
+        "macOS-10.15": { "OSVmImage": "macOS-10.15", "Pool": "Azure Pipelines" }
       },
-      "PythonVersion": "3.9",
-      "CoverageArg": ""
+      "PythonVersion": "2.7",
+      "CoverageArg": "--disablecov"
+    },
+    {
+      "CoverageConfig": {
```

Comment:
Correct me if I"m wrong but why are we limiting the windows to only the code coverage configuration?

Reply:
With the way things have been set up in all the configs across the repos, code coverage appears on a single platform. I'm not sure what the underlying reason for this is, perhaps @seankane-msft has some context here.

Reply:
Code coverage can only happen once or it will overwrite itself each time, we chose the Python 3.9 tests to run code coverage.

Reply:
This switches coverage to be in windows 3.6, do you foresee any issues with that?